odoo/http.py

```diff
@@ -1049,6 +1052,9 @@ def _default_values(self):
         self.setdefault("login", None)
         self.setdefault("session_token", None)
         self.setdefault("context", {})
+        # Force tests debug mode when test mode enabled to load 'assets_tests'
+        test_mode = odoo.tools.config['test_enable'] or odoo.tools.config['test_file']
+        self.setdefault("debug", 'tests' if test_mode else False)
```

Comment:
You would most likely need to adapt the logic of the [DisableCacheMiddleware](https://github.com/odoo/odoo/blob/e8b69e5adce0447e1880754c684bebb7e963ce2c/odoo/http.py#L1251-L1268) too